resume-buddy/src/main/java/com/google/sps/servlets/BlobstoreServeServlet.java

```diff
@@ -25,20 +23,32 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Gets the logged in user's email, finds the PDF blob and serves it
+    // Gets the logged in user's email, finds the PDF blob from the Match datastore and serves it
     UserService userService = UserServiceFactory.getUserService();
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     FetchOptions fetchOptions = FetchOptions.Builder.withLimit(Integer.MAX_VALUE);
     String email = userService.getCurrentUser().getEmail();
-    Query query = new Query("Reviewee");
-    Filter emailFilter = new FilterPredicate("email", FilterOperator.EQUAL, email);
-    query.setFilter(emailFilter);
-    PreparedQuery results = datastore.prepare(query);
-    List<Entity> entityList = results.asList(fetchOptions);
-    // TODO: (sesexton) Create a function that will grab the most recent instead of the first entity
-    // returned
-    String userBlobKeyString = entityList.get(0).getProperty("resumeBlobKey").toString();
-    BlobKey userBlobKey = new BlobKey(userBlobKeyString);
-    blobstoreService.serve(userBlobKey, response);
+
+    Query revieweeQuery = new Query("Match");
```

Comment:
I have some suggestions below to make the code a little more readable and less repetitive.